static/app/views/projectInstall/createProject.tsx

```diff
@@ -348,6 +403,16 @@ function CreateProject() {
             {...alertFrequencyDefaultValues}
             platformLanguage={platform?.language as SupportedLanguages}
             onChange={updatedData => setAlertRuleConfig(updatedData)}
+            notificationProps={{
+              actions: alertNotificationActions,
+              channel,
+              integration,
+              provider,
+              setActions: setAlertNotificationActions,
+              setChannel,
+              setIntegration,
+              setProvider,
+            }}
```

Comment:
If you make my change suggested here https://github.com/getsentry/sentry/pull/79194/files#r1805048688 could this just become

```suggestion
            notificationProps={notificationProps}
```